src/sidebar/components/button.js

```diff
@@ -55,6 +59,7 @@ export default function Button({
         className
       )}
       onClick={onClick}
+      aria-label={title}
```

Comment:
I'm not keen on always setting `aria-label` to `title`. Unlike `title`, `aria-label` will override the visible anchor text. While we want to do this for the specific case of the "More"/"Less" buttons, I don't think we want to do this in general.

For example, in VoiceOver this changes the way the group menu's menu button is read from "<Group name>. <Pause> Select group. <Interaction advice...>" to "<Select group>. <Pause> <Interaction advice...>". As a result, you can no longer tell which group is selected without opening the menu.